resolwe_bio/tests/workflows/test_wes.py

```diff
@@ -93,3 +96,16 @@ def test_wes(self):
         self.run_process('workflow-wes', input_workflow)
         wes = Data.objects.last()
         self.assertFileExists(wes, 'vcf')
+
+        # Test skipping bamclipper
+        input_workflow_skipbc = input_workflow
+        input_workflow_skipbc['advanced']['bamclipper']['skip'] = True
+        self.run_process('workflow-wes', input_workflow_skipbc)
+        wes_skip = Data.objects.filter(process__slug='bamclipper')
+
+        # wes_skip returns two objects. Find the one with the highest ID (which
```

Comment:
This can probably be solved by `wes_skip = Data.objects.filter(process__slug='bamclipper').last()`

Reply:
@romunov have use tried using `.last()`? It would be more concise.

Reply:
We're not interested in the last object as bamclipper is the second step (+ the uploads).

Reply:
```
wes_skip = Data.objects.filter(process__slug='bamclipper').last()
self.assertEqual(wes_skip.process_info, ['Skipping bamclipper step.'])
```
Test pass OK for me. 
Using `process__slug` filter will return 2 object, and `.last` will return the object with higher ID, making the code below unnecessary.